app/assets/stylesheets/config/_spaces.css

```diff
@@ -0,0 +1,64 @@
+/* paddings */
+.p-top-s {
+  padding-top: 15px;
```

Comment:
**suggestion**: Should we extract these pixel sizes to variables? Also should we consider transforming these in `rem`?